napari/layers/shapes/shape_models/shape.py

```diff
@@ -221,6 +221,7 @@ def edge_color(self, edge_color):
         else:
             rgb = tuple([int(255 * x) for x in self._edge_color.rgba[:3]])
             self._edge_color_name = '#%02x%02x%02x' % rgb
+        self._edge_color = self._edge_color.rgba
```

Comment:
can you revert this change - i think it is causing problems with the colors in the shapes layer right now, and we'll have a new PR after this just focused on shapes

Reply:
Not sure what was I thinking...